### PR TITLE
(#1266) New scalar for text ending predicate

### PR DESCRIPTION
**Pull request by fanifieiev**
This PR is for issue [1266](https://github.com/yegor256/cactoos/issues/1266)

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1269](https://github.com/yegor256/cactoos/issues/1269) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1269](https://github.com/yegor256/cactoos/issues/1269) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/2dc680b7-6e40-44bb-b7d3-7a06b5f23198); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by fanifieiev** [comment]
@fabriciofx Please review

**Comment by fanifieiev** [comment]
@fabriciofx Please review.
@paulodamaso Is there a way to reassign the reviewer????

**Comment by paulodamaso** [comment]
@0crat assign me

File: src/main/java/org/cactoos/scalar/SumOfScalar.java [hunk 1]
```diff
@@ -39,6 +39,10 @@
  * <p>Note this class is for internal usage only
  *
  * @since 0.30
+ * @todo #1266:30min This class uses Java8 Stream API. Refactor the
```
Comment (paulodamaso) [inline src/main/java/org/cactoos/scalar/SumOfScalar.java]: @fanifieiev Looks like this todo is not related to the original issue. You should file a new bug for it

File: src/main/java/org/cactoos/text/EndsWith.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Scalar;
+import org.cactoos.Text;
+
+/**
+ * Determines if the provided origin ends with the provided origin pattern.
```
Comment (paulodamaso) [inline src/main/java/org/cactoos/text/EndsWith.java]: @fanifieiev I think we could use the description like this: 'Tests if this Text ends with the specified suffix.'

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/1269#issuecomment-579016790))

@paulodamaso There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@542cb6aa2895, thread: PQ-C63314D6Z
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [comment, job, issue, repo, cause, _parent_sig, flow, login]
```

[0.51.0](https://github.com/zerocracy/farm): CID: [b20aae8d-25e7-42a6-9b58-28391f2f6441](https://www.0crat.com/footprint/C63314D6Z/b20aae8d-25e7-42a6-9b58-28391f2f6441), Type: "Order was canceled"

File: src/main/java/org/cactoos/text/EndsWith.java (hunk 2 above)
Comment (fanifieiev) [reply src/main/java/org/cactoos/text/EndsWith.java]: @paulodamaso Agree

**Comment by fanifieiev** [comment]
> @fanifieiev Seems good, just two comments please take a look

@paulodamaso Please review again.

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/cactoos/issues/1269#issuecomment-579016790))

@paulodamaso This pull request [#1269](https://github.com/yegor256/cactoos/issues/1269) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C63314D6Z/017e8a18-4f1f-4b88-b68f-af94df462f04); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 22 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)
